tidy3d/plugins/autograd/README.md

```diff
@@ -162,17 +226,17 @@ Next on our roadmap (targeting 2.8 and 2.9, fall 2024) is to support:
 
 ### Finally
 
-If you have feature requests or questions, please feel free to file an issue or discussion on the `tidy3d` front end repository.
+If you have feature requests or questions, please feel free to file an issue or discussion on the `tidy3d` front-end repository.
```

Comment:
I was going to say to add a link but we're already here, so maybe "this `tidy3d` front-end repository"?